script/Deployer.sol

```diff
@@ -77,6 +77,7 @@ contract Deployer is Script {
 
         // Wire aggregator
         aggregator.file("routers", routers);
+        aggregator.rely(address(gateway));
```

Comment:
Required now because it's changed from an `onlyGateway` modifier to an `auth` modifier